Casks/font-noto-sans-mono.rb

```diff
@@ -1,48 +1,19 @@
 cask "font-noto-sans-mono" do
-  version :latest
-  sha256 :no_check
+  version "2.014"
+  sha256 "090cf6c5e03f337a755630ca888b1fef463e64ae7b33ee134e9309c05f978732"
 
-  url "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansMono-unhinted.zip",
-      verified: "noto-website-2.storage.googleapis.com/"
-  name "Noto Sans Mono"
-  homepage "https://www.google.com/get/noto/#sans-mono"
+  url "https://github.com/notofonts/latin-greek-cyrillic/releases/download/NotoSansMono-v#{version}/NotoSansMono-v#{version}.zip",
+      verified: "github.com/notofonts/"
+  name "Noto Sans"
+  desc "Monospaced variable font"
+  homepage "https://notofonts.github.io/"
 
-  font "NotoSansMono-Black.ttf"
-  font "NotoSansMono-Bold.ttf"
-  font "NotoSansMono-Condensed.ttf"
-  font "NotoSansMono-CondensedBlack.ttf"
-  font "NotoSansMono-CondensedBold.ttf"
-  font "NotoSansMono-CondensedExtraBold.ttf"
-  font "NotoSansMono-CondensedExtraLight.ttf"
-  font "NotoSansMono-CondensedLight.ttf"
-  font "NotoSansMono-CondensedMedium.ttf"
-  font "NotoSansMono-CondensedSemiBold.ttf"
-  font "NotoSansMono-CondensedThin.ttf"
-  font "NotoSansMono-ExtraBold.ttf"
-  font "NotoSansMono-ExtraCondensed.ttf"
-  font "NotoSansMono-ExtraCondensedBlack.ttf"
-  font "NotoSansMono-ExtraCondensedBold.ttf"
-  font "NotoSansMono-ExtraCondensedExtraBold.ttf"
-  font "NotoSansMono-ExtraCondensedExtraLight.ttf"
-  font "NotoSansMono-ExtraCondensedLight.ttf"
-  font "NotoSansMono-ExtraCondensedMedium.ttf"
-  font "NotoSansMono-ExtraCondensedSemiBold.ttf"
-  font "NotoSansMono-ExtraCondensedThin.ttf"
-  font "NotoSansMono-ExtraLight.ttf"
-  font "NotoSansMono-Light.ttf"
-  font "NotoSansMono-Medium.ttf"
-  font "NotoSansMono-Regular.ttf"
-  font "NotoSansMono-SemiBold.ttf"
-  font "NotoSansMono-SemiCondensed.ttf"
-  font "NotoSansMono-SemiCondensedBlack.ttf"
-  font "NotoSansMono-SemiCondensedBold.ttf"
-  font "NotoSansMono-SemiCondensedExtraBold.ttf"
-  font "NotoSansMono-SemiCondensedExtraLight.ttf"
-  font "NotoSansMono-SemiCondensedLight.ttf"
-  font "NotoSansMono-SemiCondensedMedium.ttf"
-  font "NotoSansMono-SemiCondensedSemiBold.ttf"
-  font "NotoSansMono-SemiCondensedThin.ttf"
-  font "NotoSansMono-Thin.ttf"
+  livecheck do
+    url "https://github.com/notofonts/latin-greek-cyrillic.git"
```

Comment:
all of these should be adjusted to

```suggestion
    url :url
```

Sorry for the back and forth @sharpenedblade

Reply:
Fixed